core/templates/pages/exploration-editor-page/translation-tab/state-translation/state-translation.component.ts

```diff
@@ -574,6 +577,43 @@ export class StateTranslationComponent
     return translatableContents;
   }
 
+  getActiveTab(): string {
+    this.initActiveContentId = this.stateEditorService.
+      getInitActiveContentId();
+    if (!this.initActiveContentId) {
+      return null;
```

Comment:
Why returning `null`?

Reply:
If activeContentId is null, that means we don't have to navigate to any state, which would result to default case.

In line 659, we are performing a check for activetabtype, if it's null or disabled, we perform default navigation like we do it presently.